res/layout/files.xml

```diff
@@ -20,39 +20,21 @@
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/drawer_layout"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:clickable="true" >
+    android:layout_height="match_parent">
 
-    <!-- The main content view -->
     <LinearLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/background_color"
-        android:baselineAligned="false"
-        android:orientation="horizontal"
-        android:id="@+id/ListLayout"
-        android:contentDescription="@string/list_layout"
+        android:orientation="vertical"
         >
 
+        <!-- toolbar -->
+        <include layout="@layout/toolbar" />

```

Comment:
OK, you are not the first that suggests something like this, but I would like to know a bit about why.

FileDisplayActivity is receiving a theme without action bar, and instead a toolbar is explicitly added. Why? Is there something that we want to do with the tool bar that cannot be done with the default action bar?